pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -619,6 +619,37 @@ type KubeAPIServerConfig struct {
 	// Defaults to 1h.
 	// +optional
 	EventTTL *metav1.Duration `json:"eventTTL,omitempty" protobuf:"bytes,12,opt,name=eventTTL"`
+	// AccessControl provides authorization mechanisms for Shoot-API-Server.
+	// +optional
+	AccessControl *AccessControl `json:"accessControl,omitempty" protobuf:"bytes,13,opt,name=accessControl"`
+}
+
+// AccessControl provides authorization mechanisms for Shoot-API-Server.
+// Note: The schema (incl. child structs) and documentation resembles istio's AuthorizationPolicy.
+type AccessControl struct {
+	// The action to take on the source of request.
+	Action *AuthorizationAction `json:"action" protobuf:"bytes,1,opt,name=authorizationAction"`
```

Comment:
Why do you use a different name for protobuf?
```suggestion
	Action *AuthorizationAction `json:"action" protobuf:"bytes,1,opt,name=action"`
```